production/helm/loki/templates/write/statefulset-write.yaml

```
@@ -101,9 +102,17 @@ spec:
             {{- toYaml .Values.loki.containerSecurityContext | nindent 12 }}
           readinessProbe:
             {{- toYaml .Values.loki.readinessProbe | nindent 12 }}
-          {{- with .Values.write.lifecycle }}
+          {{- if .Values.write.lifecycle }}
           lifecycle:
-            {{- toYaml . | nindent 12 }}
+            {{- toYaml .Values.write.lifecycle | nindent 12 }}
+          {{- else }}
+            {{- if .Values.write.autoscaling.enabled }}
```

Not a big deal, but this could be a single `else if`.
```suggestion
          {{- else if .Values.write.autoscaling.enabled }}
```